content/java/concepts/calendar/terms/getActualMaximum/getActualMaximum.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '.getActualMaximum()'
+Description: 'A Calendar class method that returns the maximum value the specified calendar field could have, based on the time value of the calendar'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.getActualMaximum()`** method of the `Calendar` class retrieves the maximum value the specified calendar field could have, based on the time value of the calendar.
+
+## Syntax
+
+```pseudo
+calendar.getActualMaximum(int field)
+```
+
+## Parameters
+
+`int field`: The calendar field the maximum value is to be retrieved from. For example `Calendar.MONTH` and `calendar.YEAR`
+
+## Example
+
+In the following example, `.getActualMax()` is used to retrieve the maximum value of both the `Calendar.MONTH` and `Calendar.YEAR` fields.
+
+```java
+import java.util.*;
+
+public class CalendarMaxDemo {
+   public static void main(String[] args) {
+
+        //Creating a calendar instance 
+      Calendar calendar = Calendar.getInstance();
+
+        //Retrieving the maximum value  the year field  can have   
+        int year = 
+            calendar.getActualMaximum(Calendar.YEAR);
+            system.out.println("Maximum year is: " + year);
+
+        //Retrieving the maximum value the month field can have 
+
+        int month = 
+            calendar.getActualMaximum(Calendar.MONTH);
+            system.out.println("Maximum MONTH is: " + month); 
+
+   }
+}
```

Comment:
```suggestion
import java.util.Calendar;

 // CalendarMaxDemo.java
 public class CalendarMaxDemo {              
  public static void main(String[] args) {

  // Creating a calendar instance 
  Calendar calendar = Calendar.getInstance();

  // Retrieving the maximum value  the year field  can have   
  int year =  calendar.getActualMaximum(Calendar.YEAR);
  System.out.println("Maximum year is: " + year);

  // Retrieving the maximum value the month field can have 
    
  int month = calendar.getActualMaximum(Calendar.MONTH);
  System.out.println("Maximum MONTH is: " + month); 
  }
}
```

Reply:
There should always be one space before the start of a comment and docs code blocks must use 2 spaces, not tabs

Reply:
@caupolicandiaz Thank you for your insight. I reviewed my entry and made changes in accordance to yout comments. I changed the descpription of the method as it was truly identical to the geekforgeeks one. I also created a new example for the entry. Thanks

Reply:
Also i don't know why my commit has 2 authors yet I commited. Could you please help me out. Thanks